test/parallel/test-spawn-cmd-named-pipe.js

```diff
@@ -42,10 +42,8 @@ if (!process.argv[2]) {
     assert.fail('Failed to get COMSPEC');
   }
 
-  const args = ['/c', process.execPath, __filename, 'child',
-                '<', stdinPipeName, '>', stdoutPipeName];
-
-  const child = spawn(comspec, args);
+  const child = exec(`"${process.execPath}" "${__filename}" child < ${
```

Comment:
I've tried and failed. It seems the args are re-quoted internally in an inappropriate way for this case. But I may miss some variants. Can anybody else test?